### PR TITLE
Propagate Slack launch errors

### PR DESCRIPTION
**Pull request by lasryaric**

### CODE REVIEW

File: connectors/src/connectors/slack/temporal/client.ts [hunk 1]
```diff
@@ -36,11 +36,6 @@ export async function launchSlackSyncWorkflow(
   } catch (e) {
     return new Err(e as Error);
   }
-
```
Comment (spolu) [inline connectors/src/connectors/slack/temporal/client.ts:39]: I think we want to keep it and move the return no?

**Review by spolu — CHANGES_REQUESTED** [review]

File: connectors/src/connectors/slack/index.ts [hunk 2]
```diff
@@ -33,17 +33,15 @@ export async function createSlackConnector(
       if (teamInfo.ok !== true) {
         return new Err(
           new Error(
-            `Could not get slack team info. Error message: ${
-              teamInfo.error || "unknown"
+            `Could not get slack team info. Error message: ${teamInfo.error || "unknown"
```
Comment (spolu) [inline connectors/src/connectors/slack/index.ts:36]: not sure this will be accepted by eslint?

**Review by spolu — DISMISSED** [review]

File: connectors/src/connectors/slack/index.ts (hunk 2 above)
Comment (lasryaric) [reply connectors/src/connectors/slack/index.ts:36]: eslint does not seem to complain.

Comment (spolu) [reply connectors/src/connectors/slack/index.ts]: https://github.com/dust-tt/dust/actions/runs/4832676037/jobs/8611746779?pr=334 ?

**Review by spolu — APPROVED** [review]

**Pull request merged**